### DISCUSSION TITLE
Is that possible to setup openCommand for users of Intellij?

### DISCUSSION THREAD

**Discussion by Tim-Cao** [jesseduffield/lazygit#2192] [Q&A] [answered]
There is an available openCommand for users of VScode.
```
os:
  openCommand: 'code -rg {{filename}}'
```
Do we also support Intellij?

**Comment by mark2185** [comment, marked as answer]
> ### Recommended Config Values
>
>for users of VSCode
>
>```yaml
>os:
>  openCommand: 'code -rg {{filename}}'
>```

That's why it is in the config, so one can configure it. You can replace the `openCommand` with e.g. `vim {{filename}}` or `emacs {{filename}}`, so feel free to try replacing it with `idea {{filename}}`.

**Reply by Tim-Cao** [reply]
Many thanks @mark2185, **idea.sh {{filename}}** works for me
